map/routing_manager.cpp

```diff
@@ -1158,8 +1158,8 @@ bool RoutingManager::GenerateRouteAltitudeChart(uint32_t width, uint32_t height,
                                                 geometry::Altitudes const & altitudes,
                                                 vector<double> const & routePointDistanceM,
                                                 vector<uint8_t> & imageRGBAData,
-                                                int32_t & minRouteAltitude,
-                                                int32_t & maxRouteAltitude,
+                                                int32_t & totalAscent,
+                                                int32_t & totalDescent,
```

Comment:
If totalAscent/Descent can't be negative, please use size_t for it.

Reply:
I used uint32_t instead size_t